Update zh_cn.json

1mtwtfss1 opened the pull request.

1mtwtfss1 committed:
Update zh_cn.json

1mtwtfss1 committed:
Update zh_cn.json

Lykrast commented:
Haven't finished that upcoming update yet (real life delaying it a bit) so don't know if I won't be adding more lang entries. If I don't I'll merge this before release (and hope I don't forget).

Also lots of curios description were updated in english to have numbers (like with `%1$d%%`) and I don't see those here (though I don't blame you for not noticing).

1mtwtfss1 commented:
> Haven't finished that upcoming update yet (real life delaying it a bit) so don't know if I won't be adding more lang entries. If I don't I'll merge this before release (and hope I don't forget).
> 
> Also lots of curios description were updated in english to have numbers (like with `%1$d%%`) and I don't see those here (though I don't blame you for not noticing).

OK, I will check it again soon.

1mtwtfss1 committed:
Merge branch 'Lykrast:master' into master

1mtwtfss1 committed:
Update zh_cn.json

1mtwtfss1 commented:
> Haven't finished that upcoming update yet (real life delaying it a bit) so don't know if I won't be adding more lang entries. If I don't I'll merge this before release (and hope I don't forget).
> 
> Also lots of curios description were updated in english to have numbers (like with `%1$d%%`) and I don't see those here (though I don't blame you for not noticing).

OK, I changed these numbers into things such as `%1$d%%` now and updated it.
If no more problems, I think it can be merged.
